test/test.js

```diff
@@ -24,4 +24,12 @@ function testOrder () {
 }
 
 testConfigFile()
-testOrder()
+testOrder().catch((e) => {
+  if (process.env.DEBUG) {
+    console.error(e.message)
+  } else {
+    console.error(e.name)
+    console.error('Run with the DEBUG environement variable to see more.')
+  }
+  process.exit(-1)
+})
```

Comment:
Without this change, `npm test` doesn't fail if there's a failure, which is unfortunate :-)
I also decided to not show the full message by default because it's painful. But I can change this if you prefer. Ideally we should make a diff of expected vs actual, but this is more than this PR is about and didn't want to do it now. I could look into it if you want...